name-segments/nouns/urban.txt

```diff
@@ -0,0 +1,17 @@
+metro
+metropolitan
+urbanite
+city
+urbano
+citylights
+streetsahead
+road
+skyhigh
+skyscrapers
```

Comment:
The issue #25 , is specifically looking for Fantasy-based nouns. That is, the type of stories that dragons and princesses live in. There are no skyscrapers here